IsogeomGenerator/generate_isogeom.py

```diff
@@ -0,0 +1,356 @@
+import argparse
+import os
+import vol as v
+
+
+def set_level_options(parser, moab):
+    """Sets options for specifying level values.
+
+    Three options are available (two available for MOAB mode). Exactly one
+    option must be supplied. If in MOAB mode (moab=True), do not give options
+    for generating levels.
+
+    Input:
+    ------
+        parser: ArgumentParser object to attach options to
+        moab: bool, True if setting MOAB args
+    """
+    level_group = parser.add_mutually_exclusive_group(required=True)
```

Comment:
Question for reviewers (@bam241 @ypark234 @nuclearkatie): Right now, with this "mutually exclusive group" there is no true indication in the help message that indicates 1) that it is mutually exclusive or 2) that it is required. The `add_mutually_exclusive_group` in argparse does not support extra information like a description or title. However, there is the `add_group` which does support a description and title, but obviously does not check that at least one of the group is set. So my question is, which would you do:

1. used `add_mutually_exclusive_group` so that it automatically checks that one of the three options is set. But then include a description in the main parser argument that one of the three options must be set.

or

2. Use `add_group` to create a separate group where there can be a formatted description about needing to supply one of the three arguments and why. But I would have to make a separate check that one of them was supplied since they would all need to be optional arguments.

thoughts?

Reply:
Right now, I opted for information in the description of the parser.

Reply:
Looks good to me